### PR TITLE
Add new page for oolleegg55

### PR DESCRIPTION
**Pull request by oolleegg55**
New page from oolleegg55. Look it, please.

### CODE REVIEW

**Comment by oolleegg55** [comment]
@githubteacher The travis-ci checked the other files in my pull request and found error there. But my file has no error. What did I do incorrect?

File: _pages/oolleegg55.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: oolleegg55
```
Comment (leiguoyang) [inline _pages/oolleegg55.yaml]: `githubhandle` to `githubHandle` i think.

File: _pages/oolleegg55.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+pageURL: oolleegg55.github.io
```
Comment (leiguoyang) [inline _pages/oolleegg55.yaml]: @oolleegg55 I think `pageURL` is supposed to be `pageUrl` as in the course.

File: _pages/oolleegg55.yaml (hunk 1 above)
Comment (oolleegg55) [inline _pages/oolleegg55.yaml]: @leiguoyang Sorry, can you explain me how do you know it? Why travic didn't show it? Or I missed it?

File: _pages/oolleegg55.yaml (hunk 2 above)
Comment (leiguoyang) [inline _pages/oolleegg55.yaml]: Sorry, @oolleegg55  I don't understand the travis-ci message, either.  I see some error message related to other yaml files, but cannot find yours.

I encounter the same problem too in my pull request and want to figure it out.

@githubteacher Could you teach us how to read the travis-ci message?

Comment (oolleegg55) [inline _pages/oolleegg55.yaml]: @leiguoyang I suggest to wait answer. May be we missed something.

**Pull request merged**